apps/src/studio/blocks.js

```diff
@@ -962,17 +940,32 @@ exports.install = function(blockly, blockInstallOptions) {
     helpUrl: '',
     init: function() {
       var dropdown = new blockly.FieldDropdown(this.VALUES);
+      var dropdown1 = new blockly.FieldDropdown(
```

Comment:
as a general rule, you should always avoid naming variables by just adding on counts. I'd call this something like

```suggestion
      var spriteIndexDropdown = new blockly.FieldDropdown(
```